python/cudf/cudf/_lib/pylibcudf/groupby.pyx

```diff
@@ -253,8 +253,8 @@ cdef class GroupBy:
         Parameters
         ----------
         values : Table, optional
-            The columns to get group labels for. If not specified, the group
-            labels for the group keys are returned.
+            The columns to get group labels for. If not specified,
+            an empty table is returned for the group values.
```

Comment:
Should we return an empty table or should we not return anything at all? On one hand I tend to prefer the return type not changing based on arguments, on the other I would prefer not creating an extra object for no reason and there is precedent for this kind of thing in numpy APIs (like np.unique).

Reply:
No preference. I changed t oreturn `None`.

Reply:
Also saved a reference to keys table.